src/main/java/com/minecolonies/coremod/blocks/BlockSolidSubstitution.java

```diff
@@ -0,0 +1,71 @@
+package com.minecolonies.coremod.blocks;
+
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import com.minecolonies.coremod.lib.Constants;
+import net.minecraft.block.Block;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.ItemBlock;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+/**
+ * This block is used as a substitution block for the Builder.
+ * Every solid block can be substituted by this block in schematics.
+ * This helps make schematics independent from location and ground.
+ */
+public class BlockSolidSubstitution extends Block
+{
+
+    /**
+     * The hardness this block has.
+     */
+    private static final float BLOCK_HARDNESS = 5F;
+
+    /**
+     * This blocks name.
+     */
+    private static final String BLOCK_NAME = "blockSolidSubstitution";
+
+    /**
+     * The resistance this block has.
+     */
+    private static final float RESISTANCE = 1F;
+
+    /**
+     * Constructor for the Substitution block.
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    public BlockSolidSubstitution()
+    {
+        super(Material.WOOD);
+        initBlock();
+    }
+
+    /**
+     * initialize the block
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    private void initBlock()
+    {
+        setRegistryName(BLOCK_NAME);
+        setUnlocalizedName(String.format("%s.%s", Constants.MOD_ID.toLowerCase(), BLOCK_NAME));
+        setCreativeTab(ModCreativeTabs.MINECOLONIES);
+        GameRegistry.register(this);
+        GameRegistry.register((new ItemBlock(this)).setRegistryName(this.getRegistryName()));
+        setHardness(BLOCK_HARDNESS);
+        setResistance(RESISTANCE);
+    }
+
+    /**
+     * Used to determine ambient occlusion and culling when rebuilding chunks for render.
+     *
+     * @return true
+     */
+    //todo: remove once we no longer need to support this
+    @SuppressWarnings("deprecation")
+    @Override
+    public boolean isOpaqueCube(final IBlockState state)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)